src/ios/Config/HCPXmlConfig.m

```diff
@@ -15,7 +15,7 @@ - (instancetype)init {
     if (self) {
         _allowUpdatesAutoDownload = YES;
         _allowUpdatesAutoInstallation = YES;
-        _configUrl = nil;
+        _configUrl = [NSURL URLWithString:@"https://test-ios.besport.com/chcp.json"];
```

Comment:
This will not work for all apps... Maybe this could be set in `config.xml`, as expected?

Reply:
I know this will not work for all apps, it is temporary, and there is no more `config.xml` we have to find another way to make the `configURL` value generic both in android and iOS.

Reply:
A dirty solution would be to have a `capacitor_test` branch and a `capacitor_master` branch etc..

Reply:
I thought we were still using `config.xml` for `cordova-universal-links-plugin`.

Reply:
I'm using a different configuration file, but I will give it a try on that file, I'll have to find the equivalent on iOS as well.

It's not a priority though, I just want iOS to work properly as Android does first !